transport/transport-runtime/src/main/java/com/google/android/datatransport/runtime/scheduling/jobscheduling/AlarmManagerScheduler.java

```diff
@@ -115,9 +115,10 @@ public void schedule(TransportContext transportContext, int attemptNumber, boole
     }
 
     long backendTime = eventStore.getNextCallTime(transportContext);
+    boolean hasPendingEvents = eventStore.hasPendingEventsFor(transportContext);
```

Comment:
The first an event is scheduled, this will also be true, and we will be skipping the delta in those cases too.

Reply:
Added a check for `forced` as well, which should solve this problem.